### PR TITLE
fix: updates for print proxy v2

### PR DESCRIPTION
**Pull request by stdavis**
Waiting on the production release of Print Proxy v2 for these changes.

### CODE REVIEW

**Review by steveoh — APPROVED** [review]

File: src/data/contacts.ts [hunk 1]
```diff
@@ -419,11 +419,6 @@ export const contacts: Contacts = {
     email: 'SteveBowman@utah.gov',
     phone: '801-537-3304',
   },
-  print_proxy: {
-    name: 'Scott Davis',
-    agency: 'UGRC',
-    email: 'stdavis@utah.gov',
-  },
```
Comment (steveoh) [inline src/data/contacts.ts:426]: what was the reasoning to pull out of here?

Comment (stdavis) [reply src/data/contacts.ts:426]: Because I had everything going to ugrc-developers@utah.gov for print proxy.

Comment (steveoh) [reply src/data/contacts.ts:426]: Are you ok with using the contact component and updating the contacts to point to the generic ugrc developers? We have a "rule" not to have mailto links in pages. 

I think we should consolidate our work emails for the group in other places. I can put together that pull request if you're happy with that.

Comment (stdavis) [reply src/data/contacts.ts:426]: I didn't realize that the mailto rule applied to generic accounts but I'm good with that. I'll submit a quick PR to fix this page.

Comment (steveoh) [reply src/data/contacts.ts:426]: thanks. https://github.com/agrc/gis.utah.gov#contact-information

**Pull request merged**